packages/E2ETest/wdio/test/Image.spec.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import HomePage from '../pages/HomePage';
+import ImageTestPage from '../pages/ImageTestPage';
+import assert from 'assert';
+
+beforeAll(() => {
+  HomePage.backToHomePage();
+  HomePage.clickAndGotoImagePage();
+});
+
+describe('ImageWithoutBorderTest', () => {
+  it('ImageWithoutBorderTest Success', () => {
+    const result = ImageTestPage.getTreeDumpResult();
+    assert.ok(
+      result.includes('TreeDump:Passed'),
+      'Dump comparison passed for image without border!'
+    );
```

Comment:
If the test fails, will anything be visible in the test output or as test artifacts other than this message?  This will be particularly important when the test is running in the CI loop.  Note this doesn't have to be done for this checkin - feel free to file an issue to improve this.

Reply:
Added steps to generate artifacts for the generated .out file and .err file during the test.

---
In reply to: [356351609](https://github.com/microsoft/react-native-windows/pull/3754#discussion_r356351609) [](ancestors = 356351609)